### ISSUE TITLE
Collaboration Cycle for  [DGIB, Secure Inbox/Messaging (for Education)]

### ISSUE THREAD

**Issue by cesaresantiago** [department-of-veterans-affairs/va.gov-team#50481]
### If your product is already on VA.gov, please verify your product information in the Product Directory.

- [X] The product information in the Product Directory is accurate.

### VFS team name

DGIB

### Product name

Secure Inbox/Messaging

### Feature name

Secure Inbox/Messaging

### GitHub label for product

afs-dgib

### When did/will you start working on this product/feature?

01/01/2022

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Tools, applications and dynamic pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

No

### Do you need to capture any additional analytics or metrics?

No

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/my-education-benefits/VA%20Education%20Inbox/product-outline.md

### Notify the Collaboration Cycle team of this ticket in the vfs-platform-support Slack channel.

- [X] I acknowledge that I must notify \#vfs-platform-support after submitting this issue.

**Comment by cesaresantiago** [comment]
The collaboration kickoff for this application was already started on the older template: https://github.com/department-of-veterans-affairs/va.gov-team/issues/29756

Midpoint reviews also took place.

We want to move forward to staging review. cc: @shiragoodman

**Comment by shiragoodman** [comment]
@cesaresantiago @btruongafs When you completed this ticket, you check the box:

> [X] The product information in the Product Directory is accurate.

However, it appears it is not as the name is still [Secure Inbox.](https://depo-platform-documentation.scrollhelp.site/getting-started/secure-inbox) Can you please verify the information we have and [submit a ticket](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory-user-guide) for updates?

**Comment by cesaresantiago** [comment]
@shiragoodman : that would be because when I filled the form i opened it against "Secure Inbox" and only later changed the name. I will do the request  to change it.

**Comment by cesaresantiago** [comment]
Change requested here: https://github.com/department-of-veterans-affairs/product-directory/issues/572

cc: @shiragoodman  @btruongafs

**Comment by btruongafs** [comment]
Ticket submitted for updates: https://github.com/department-of-veterans-affairs/product-directory/pull/579

**Comment by bethpottsVADEPO** [comment]
@btruongafs
I want to make sure that ahead of the staging reviews next week for education benefit letters this ticket is up to date with a link to the [static Drupal page](http://preview-prod.vfs.va.gov/preview?nodeId=51823) the content team created. Would you like the content team to add to the artifacts section above or would you like to make this update to the top section of the ticket? The Platform team will need to review this Drupal page, not the HTML static page currently in staging that has outdated content at this point.

**Comment by btruongafs** [comment]
@bethpottsVADEPO yes please do update the ticket with the content artifacts. appreciate it!

**Comment by btruongafs** [comment]
For Staging Reviewers:

Since Design and Midpoint Review completion, the original design was scaled back. MVP will only display only 1 Decision Letter (most recent). 
Updated mockup/prototype design: https://app.abstract.com/share/4153d410-1183-4818-9c69-8c338c526b1b

**Comment by sterkenburgsara** [comment]
The staging review for the static Drupal page (sitewide content & IA team) scheduled for 1/20/22 was cancelled as 0 issues were identified.

**Comment by joshkimux** [comment]
Hi @btruongafs 🌻 

Small question for you and team-- I couldn't find it in the ticket or in the research repo-- but did this product ever get usability testing? Context: we're trying to look into the a11y health of products across VA.gov.

**Comment by joshkimux** [comment]
@artsymartha68 for awareness
